Feature/multiple schema

DanteUkraine opened the pull request:
Beta is available for testing:
npm i playwright-graphql@2.0.0-beta2

This release brakes backwards compatibility. But introduces new version of CLI that does not require codegen files.
I added options --saveCodegen and --custom for those users who need advance configurability.
Custom mode will not validate codegen so users are not limited.


For users who used CLI without modifying codegen, migration will be seamless.
For those who use custom operations and manually modified codegen now they have 2 options:

 1. New CLI support adding custom operations:
 `playwright-graphql --url http://localhost:4000/api/graphql --document src/operations`
and this can be combined with introspect flag. No need to maintain codegen.ts.  
`playwright-graphql --url http://localhost:4000/api/graphql --document src/operations --introspect false`

 2. Flag `--custom` allows user to run codegen form codegen file and it will not be overridden.

andrew-demb commented:
Can you provide a PR to demo application repository about required changes for ebd users, please?

DanteUkraine commented:
@andrew-demb, braking changes affect only users that use custom operations and modified codegen file manually.
yes I will provide more information and code examples as soon as I will have some capacity.

DanteUkraine commented:
@andrew-demb I added to readme file more examples how to use custom operations without codegen.ts. Users will be affected only in case they modified codegen manually because I dropped support of this file.

andrew-demb reviewed (approved):
I've just checked with the example project (simple use-case, no multiple schemas) - it works OK https://github.com/DanteUkraine/playwright-graphql-example/pull/1

I cannot review JS/readme soon. Let's release it as is.

DanteUkraine commented:
@andrew-demb I will update example project, it is in-progress.
